kitsune/sumo/views.py

```diff
@@ -26,11 +27,54 @@
 from kitsune.sumo.urlresolvers import reverse
 from kitsune.sumo.utils import get_next_url, uselocale
 from kitsune.users.forms import AuthenticationForm
+from kitsune.wiki.models import Document
 
 
 log = logging.getLogger('k.services')
 
 
+# copied from
+# https://github.com/mozilla/kuma/blob/master/kuma/health/views.py
+@require_safe
+def liveness(request):
+    """
+    A successful response from this endpoint simply proves
+    that Django is up and running. It doesn't mean that its
+    supporting services (like MySQL, Redis, Celery) can
+    be successfully used from within this service.
+    """
+    return HttpResponse(status=204)
+
+
+# copied from
+# https://github.com/mozilla/kuma/blob/master/kuma/health/views.py
+@require_safe
+@cache_page(10)
```

Comment:
I believe this endpoint should not be cached. If cached for 10 seconds, the endpoint will not hit the DB for 10 seconds.

Reply:
The health checks will be more than 10s apart, and this is an extra check to make sure the cache is working (as mentioned in the doc string).